src/modules/migration/MigrationListItem.tsx

```diff
@@ -175,109 +223,76 @@ export const MigrationListItem = ({
+                </Typography>
+              </Button>
+              <SvgIcon sx={{ px: 1.5 }}>
+                <ArrowNarrowRightIcon
+                  fontSize="14px"
+                  color={
+                    disabled === undefined ? theme.palette.text.primary : theme.palette.text.muted
+                  }
+                />
+              </SvgIcon>
+              <Button
+                variant="outlined"
+                size="small"
+                sx={{ width: '50px', background: 'white' }}
+                disabled
+              >
+                <Typography variant="buttonS" color={baseColor}>
+                  Variable
+                </Typography>
+              </Button>
+            </Box>
+          </ListColumn>
+        )}
 
-      {!!borrowApyType && (
-        <ListColumn align="right">
-          <Box sx={{ display: 'flex' }}>
-            <Button
-              variant="outlined"
-              size="small"
-              sx={{ width: '50px', background: 'white' }}
-              disabled
-            >
-              <Typography variant="buttonS" color={baseColor}>
-                {borrowApyType}
-              </Typography>
-            </Button>
-            <SvgIcon sx={{ px: 1.5 }}>
-              <ArrowNarrowRightIcon
-                fontSize="14px"
-                color={
-                  disabled === undefined ? theme.palette.text.primary : theme.palette.text.muted
-                }
-              />
-            </SvgIcon>
-            <Button
-              variant="outlined"
-              size="small"
-              sx={{ width: '50px', background: 'white' }}
-              disabled
-            >
-              <Typography variant="buttonS" color={baseColor}>
-                Variable
-              </Typography>
-            </Button>
+        <ListColumn align="right" maxWidth={150} minWidth={150}>
+          <Box sx={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-end' }}>
+            <Box sx={{ display: 'flex', alignItems: 'flex-end', mb: 0.5 }}>
+              <FormattedNumber value={amount} variant="secondary14" color={baseColor} />
+            </Box>
+            <FormattedNumber
+              value={amountInUSD}
+              variant="secondary12"
+              color={baseColor}
+              symbol="USD"
+              symbolsColor={baseColor}
+            />
           </Box>
         </ListColumn>
-      )}
+      </Box>
 
-      <ListColumn align="right" maxWidth={150} minWidth={150}>
-        <Box sx={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-end' }}>
-          <Box sx={{ display: 'flex', alignItems: 'flex-end', mb: 0.5 }}>
-            <FormattedNumber value={amount} variant="secondary14" color={baseColor} />
-          </Box>
-          <FormattedNumber
-            value={amountInUSD}
-            variant="secondary12"
-            color={baseColor}
-            symbol="USD"
-            symbolsColor={baseColor}
```

Comment:
The change on this file looks way scarier than it's, i just had to wrap everything in a `Box` since i needed to have 2 rows